src/lib/components/billing/selectPaymentMethod.svelte

```diff
@@ -0,0 +1,115 @@
+    let showPaymentModal = false;
+
+    async function cardSaved(event: CustomEvent<PaymentMethodData>) {
+        value = event.detail.$id;
+        methods = await sdk.forConsole.billing.listPaymentMethods();
+    }
+
+    onMount(() => {
+        if (methods?.total && methods.paymentMethods.some((method) => !!method?.last4)) {
+            value = methods.paymentMethods[0].$id;
+        }
+    });
+
+    $: filteredMethods = methods?.paymentMethods?.filter((method) => !!method?.last4);
+
+    $: selectedPaymentMethod = methods?.paymentMethods?.find((method) => method.$id === value);
+</script>
+
+{#if filteredMethods?.length}
+    {#if selectedPaymentMethod?.country === 'in'}
+        <Alert type="warning">
+            <svelte:fragment slot="title">Indian credit or debit card-holders</svelte:fragment>
+            To comply with RBI regulations in India, Appwrite will ask for verification to charge up
+            to $150 USD on your payment method. We will never charge more than the cost of your plan
+            and the resources you use, or your budget cap limit. For higher usage limits, please contact
+            us.
+        </Alert>
+    {/if}
+    <InputSelectSearch
+        id="method"
+        label="Payment method"
+        placeholder="Select payment method"
+        bind:value
+        options={filteredMethods.map((method) => {
+            return {
+                value: method.$id,
+                label: `${capitalize(method.brand)} ending in ${method.last4}`,
+                data: [method.brand]
+            };
+        })}
+        interactiveOutput
+        let:option={o}>
+        <svelte:fragment slot="output" let:option={o}>
+            <output class="input-text">
+                <span class="u-flex u-gap-16 u-flex-vertical">
+                    <span class="u-flex u-gap-16">
+                        <span class="u-flex u-cross-center u-gap-8" style="padding-inline:0.25rem">
+                            <span>{o.label}</span>
+                            <CreditCardBrandImage brand={o.data?.toString()} />
+                        </span>
+                    </span>
+                </span>
+            </output>
+        </svelte:fragment>
+
+        <span class="u-flex u-gap-16 u-flex-vertical">
+            <span class="u-flex u-gap-16">
+                <span class="u-flex u-cross-center u-gap-8" style="padding-inline:0.25rem">
+                    <span>{o.label}</span>
+                    <CreditCardBrandImage brand={o.data?.toString()} />
+                </span>
+            </span>
+        </span>
+        <svelte:fragment slot="listEnd">
+            <Button text on:click={() => (showPaymentModal = true)}>
+                <span class="icon-plus"></span>
+                <span class="text">Add new payment method</span>
+            </Button>
+        </svelte:fragment>
+    </InputSelectSearch>
+{:else}
+    <Card isDashed style="--p-card-padding: 0.75rem; --p-card-bg-color: transparent" isTile>
```

Comment:
instead of a style attribute, cant we use https://svelte.dev/docs/component-directives#style-props ?